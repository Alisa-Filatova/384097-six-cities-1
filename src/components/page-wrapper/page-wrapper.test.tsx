import * as React from 'react';
import * as renderer from 'react-test-renderer';
import PageWrapper from './page-wrapper';

describe('PageWrapper', () => {
  it('renders correctly', () => {
    const wrapper = renderer.create(
        <PageWrapper location={'/login'}>
          <div>Some page</div>
        </PageWrapper>
    ).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
