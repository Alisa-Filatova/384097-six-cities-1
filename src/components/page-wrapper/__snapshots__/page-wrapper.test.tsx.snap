// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageWrapper renders correctly 1`] = `
<div
  className="page page--gray page--login"
>
  <div>
    Some page
  </div>
</div>
`;
